src/engine/BMBtnSkillRandomBMPentaskill.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * BMBtnSkillRandomBMPentaskill: Code specific to RandomBMPentaskill
+ *
+ * @author: james
+ */
+
+/**
+ * This class currently supports the special skills of RandomBMPentaskill, which has
+ * 4 normal dice and a swing die, with 5 skills appearing a total of 13 times on various dice.
```

Comment:
Hmm.  It's not instantly obvious to me why 5 skills would naturally appear a total of 13 times, nor how many times i should expect each of the five skills to appear.  I can source-dive and figure out (but it didn't jump out at me and i don't want to spend my test time on this rather than actually testing), but i bet players will be curious what to expect too, so you should probably flesh out the button special description to give some more detail.